docs/docs/en/guide/installation/rainbond.md

```diff
@@ -0,0 +1,64 @@
+# Use Rainbond Deployment
```

Comment:
Please move `docs/docs/en/guide/installation/rainbond.md` to `docs/docs/en/guide/integration/rainbond.md`. The same with the Chinese docs. Other LGTM.

Reply:
ok, no problem